src/perfEvents_linux.cpp

```diff
@@ -880,7 +886,14 @@ int PerfEvents::walk(int tid, void* ucontext, const void** callchain, int max_de
 
         while (tail < head) {
             struct perf_event_header* hdr = ring.seek(tail);
+
             if (hdr->type == PERF_RECORD_SAMPLE) {
+
+                if (_record_cpu) {
+                  u64 cpu = ring.next();
+                  java_ctx->cpu = cpu;
```

Comment:
Use 4 spaces indent.
IMO, local variable is redundant here, just set `java_ctx->cpu = ring.next();`